examples/fp_multiplier/test_fp8_mul.py

```diff
@@ -0,0 +1,69 @@
+import cocotb
+from cocotb.clock import Clock
+from cocotb.runner import get_runner
+from cocotb.triggers import FallingEdge
+
+import os
+from pathlib import Path
+
+
+@cocotb.test
+async def fp8_mul_test_all(dut):
+    """
+    Exhaustive test of all inputs with verification against APyTypes.
+    """
+    from apytypes import (
+        APyFloatArray,
+        APyFloat,
+        APyFloatQuantizationContext,
+        QuantizationMode,
+    )
```

Comment:
I temporarily put the imports here, `nanobind` will otherwise report many things as "leaked".